test/TEST-61-MULTINIC/test.sh

```diff
@@ -15,7 +15,7 @@ test_check() {
         return 1
     fi
 
-    test -d /etc/sysconfig/network-scripts
+    command -v exportfs &> /dev/null
```

Comment:
I was working on making this test run against the Ubuntu package and had this exact change staged locally. :laughing: